src/sailfishui/plugin/systemdservice.cpp

```diff
@@ -154,7 +170,27 @@ void SystemdServicePrivate::propertiesChanged(const QString &interface,
     }
 }
 
-void SystemdServicePrivate::stateChanged(QDBusPendingCallWatcher *call) {
+void SystemdServicePrivate::handleUnitNew(const QString &name, const QDBusObjectPath &path)
+{
+    if (name == serviceName)
```

Comment:
For sake of consistency add bracket to this if and the one on line 181 similarly as you have in 187.